cpp/log/cert.cc

```diff
@@ -233,8 +262,8 @@ string Cert::PrintTime(ASN1_TIME* when) {
 }
 
 
-Cert::Status Cert::IsIdenticalTo(const Cert& other) const {
-  return X509_cmp(x509_, other.x509_) == 0 ? TRUE : FALSE;
+StatusOr<bool> Cert::IsIdenticalTo(const Cert& other) const {
+  return StatusOr<bool>(X509_cmp(x509_, other.x509_) == 0 ? true : false);
```

Comment:
It's nothing specific to `StatusOr`, though, if you have a function that returns `bool`, you can `return 42;` just fine (that's "true"), and we can't really say that we can avoid `int` returns. :wink: 

In other words, returning `StatusOr<Foo>` behaves pretty closely to how it would if you had defined the function as returning `Foo`, whatever that may be. And integer types in C++ convert rather merrily between each others (lest too much C compatibility was lost), so that's something we have to live with.